zh/access-dashboard.md

```diff
@@ -0,0 +1,135 @@
+---
+title: 访问 TiDB Dashboard
+summary: 介绍如何在 Kubernetes 环境下访问 TiDB Dashboard
+category: how-to
+---
+
+# TiDB Dashboard 指南
+
+TiDB Dashboard 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://github.com/pingcap-incubator/tidb-dashboard) 了解详情。本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
+
+## 快速上手
+
+> **注意：**
+>
+> 以下教程仅为演示如何快速访问 TiDB Dashboard，请勿在生产环境中直接使用以下方法。 
+
+`TiDB Dashboard` 目前在 4.0 版本中已经内嵌在了 PD 组件中，你可以通过以下的例子在 Kubernetes 环境下快速部署一个 4.0.0-rc 版本的 TiDB 集群。运行 `kubectl apply -f` 命令，将以下 yaml 文件部署到 Kubernetes 集群中。
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: basic
+spec:
+  version: v4.0.0-rc
+  timezone: UTC
+  pvReclaimPolicy: Delete
+  pd:
+    baseImage: pingcap/pd
+    replicas: 1
+    requests:
+      storage: "1Gi"
+    config: {}
+  tikv:
+    baseImage: pingcap/tikv
+    replicas: 1
+    requests:
+      storage: "1Gi"
+    config: {}
+  tidb:
+    baseImage: pingcap/tidb
+    replicas: 1
+    service:
+      type: ClusterIP
+    config: {}
+
+```
+
+当集群创建完毕时，你可以通过以下指令将 `TiDB Dashboard` 暴露在本地机器:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl port-forward svc/tidb-pd -n ${namespace} 2379:2379
+```
+
+然后在浏览器中访问 <http://localhost:2379/dashboard> 即可访问到 TiDB Dashboard。
+
+## 通过 Ingress 访问 TiDB Dashboard
+
+> **注意：**
```

Comment:
It seems unnecessary to write this passage in the **note:** format because there is only one passage between two titles.